### PR TITLE
Enhancement in port_bounce.py with multipath checks

### PR DESCRIPTION
**Pull request by Naresh-ibm**
added couple of functions which checks multipath
status, both when disable and enable of switch ports

Signed-off-by: Naresh Bannoth <nbannoth@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by Naresh-ibm** [comment]
[port_bounce_latest_log.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/1437596/port_bounce_latest_log.txt)

File: io/disk/port_bounce.py [hunk 1]
```diff
@@ -146,25 +152,29 @@ def porttoggle(self, test_ports, sleep_time):
...
+        time.sleep(sleep_time)
         # Port Enable
         self.log.info("Enable port(s) %s", test_ports)
         try:
             self.fc_run_command("portenable %s" % port_input)
         except CommandFailed as cf:
             self.log.info("port enable failed for port %s, details: %s",
                           test_ports, str(cf))
-        time.sleep(5)
+        time.sleep(20)
```
Comment (narasimhan-v) [inline io/disk/port_bounce.py]: I understand the need for waiting here. But there is no fool proof wait time. This is a good candidate for `avocado.utils.wait.wait_for`

Comment (narasimhan-v) [reply io/disk/port_bounce.py]: Also, consider moving the sleep / wait to corresponding functions instead of having here.

File: io/disk/port_bounce.py [hunk 2]
```diff
@@ -223,7 +278,6 @@ def tearDown(self):
         self.log.debug("Kernel Errors: %s", output)
         # verify given test ports are online after test.
         self.verify_port_enable(self.port_ids)
-        self.verify_port_toggle_host("Online")
```
Comment (narasimhan-v) [inline io/disk/port_bounce.py]: Why is this removed ?

**Comment by PraveenPenguin** [comment]
@Naresh-ibm any update on this

**Comment by PraveenPenguin** [comment]
closing this PR as it has conflict as well pending from long time @Naresh-ibm if still want to pursue  this PR please raise new one
